src/components/TodoCreate/TodoCreate.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState, ChangeEvent, FormEvent } from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import styled from 'styled-components/macro';
+import { ReactComponent as CirclePlus } from 'assets/svg/circle-plus.svg';
+import { ReactComponent as Calender } from 'assets/svg/calendar.svg';
+import { create } from 'context/todoContext/actionCreators';
+import { useTodosDispatch } from 'context/todoContext/TodoContext';
+import { CreatedTodo, TCategory, TPriority } from 'types';
+import Dropdown from '../common/Dropdown';
+
+const categoryEmoji = {
+  업무: '👩‍💻',
+  공부: '📚',
+  생활: '🌱',
+  운동: '🏃‍♂️',
+  기타: '💬',
+};
+
+const priorityEmoji = {
+  상: '🔴',
+  중: '🟡',
+  하: '🟢',
+};
+
+const categoryOptions: any = [
+  { print: '카테고리', data: '', emoji: '' },
+  ...Object.entries(TCategory).map(([key, value]) => {
+    return { print: value, data: key, emoji: categoryEmoji[value] };
```

Comment:
이모지도 하나의 문자열이니깐 `print` 의 값으로 한번에 보낼 수 있습니다!
```suggestion
    return { print: `${categoryEmoji[value]} ${value}`, data: key };
```
`dropdown` 에 사용된 `emoji` 관련 코드는 제거하면 좋을거같아요 👍

Reply:
아 그러네요! 그 생각을 미처 하지 못했던 것 같습니다....! ❗❕❗😥

바로 반영하겠습니다!